include/crm/msg_xml.h

```diff
@@ -23,17 +23,32 @@ extern "C" {
 /* This file defines constants for various XML syntax (mainly element and
  * attribute names).
  *
- * For consistency, new constants should start with "PCMK_", followed by "XE"
- * for XML element names, "XA" for XML attribute names, and "META" for meta
- * attribute names. Old names that don't follow this policy should eventually be
- * deprecated and replaced with names that do.
+ * For consistency, new constants should start with "PCMK_", followed by:
+ * * "XE" for XML element names
+ * * "XA" for XML attribute names
+ * * "OPT" for cluster option (property) names
+ * * "META" for meta-attribute names
+ * * "VALUE" for enumerated values for various options
```

Comment:
We need a convention for node attribute names as well. "NODE" could work, though that might overlap with something else, I haven't looked.

Reply:
Looks like there's no conflict and we've already started with `NODE` in `include/crm/common/nodes.h`. So we could continue with that in nodes.h (and a new nodes_internal.h if needed), if you want.

Reply:
Yep, `PCMK_NODE_ATTR_`. Previously we used `CRM_ATTR_` in crm.h.

Reply:
Ah yeah `NODE_ATTR`, I was thinking `NODE` since we're dropping `ATTR` everywhere else for brevity, but the result would be less clear for nodes that way.